### DISCUSSION TITLE
Does Rocket support Web Components?

### DISCUSSION THREAD

**Discussion by EAzari** [rwf2/Rocket#2756] [Ideas] [answered]
Hi there?

I'm thinking to use Rocket as frontend and [Web Components](https://developer.mozilla.org/en-US/docs/Web/API/Web_components) as backend, 
Have you done that in Rocket?

**Comment by strowbeary** [comment, marked as answer]
Hello,
To clarify, Rocket is primarily a back-end framework, while Web Components are a front-end technology. Therefore, you would typically use Rocket to handle the server-side logic and Web Components to build the user interface on the client side.
You can certainly use them together in the same project, with Rocket serving the backend and Web Components managing the frontend.
